x-pack/legacy/plugins/canvas/public/components/editor/editor.scss

```diff
@@ -0,0 +1,3 @@
+.monaco-editor textarea:focus {
+  animation: none !important; // Removes textarea EUI blue underline animation from EUI
```

Comment:
@ryankeairns This `!important` cases the sasslint to fail but without it, I think some EUI CSS attempts to apply the blue underline animation that most text inputs have. Is there a workaround?

Reply:
Perhaps there's a way to add enough specificity to get around it...?